From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Erik Torres Aguilar <ertorres@microsoft.com>
Date: Thu, 10 Dec 2020 22:02:47 -0800
Subject: MSFT-PATCH: Apply changes from:
 011-MSFT-Patch_change-tests-data-to-not-include-blocked-regions.patch


diff --git a/icu/icu4c/source/test/cintltst/uregiontest.c b/icu/icu4c/source/test/cintltst/uregiontest.c
index 93e2f19b6e753e0c21defc83d45588dc03935216..0412fa956dba311c5edc8fbd7f0e280e84fa0ee6 100644
--- a/icu/icu4c/source/test/cintltst/uregiontest.c
+++ b/icu/icu4c/source/test/cintltst/uregiontest.c
@@ -148,17 +148,14 @@ static KnownRegion knownRegions[] = {
     { "CZ" , 203, "151", URGN_TERRITORY, "150" },
     { "DD" , 276, "155", URGN_TERRITORY, "150" },
     { "DE" , 276, "155", URGN_TERRITORY, "150" },
-    { "DG" , -1 , "QO" , URGN_TERRITORY, "009" },
     { "DJ" , 262, "014", URGN_TERRITORY, "002" },
     { "DK" , 208, "154", URGN_TERRITORY, "150" },
     { "DM" , 212, "029", URGN_TERRITORY, "019" },
     { "DO" , 214, "029", URGN_TERRITORY, "019" },
     { "DZ" ,  12, "015", URGN_TERRITORY, "002" },
-    { "EA" ,  -1, "015", URGN_TERRITORY, "002" },
     { "EC" , 218, "005", URGN_TERRITORY, "019" },
     { "EE" , 233, "154", URGN_TERRITORY, "150" },
     { "EG" , 818, "015", URGN_TERRITORY, "002" },
-    { "EH" , 732, "015", URGN_TERRITORY, "002" },
     { "ER" , 232, "014", URGN_TERRITORY, "002" },
     { "ES" , 724, "039", URGN_TERRITORY, "150" },
     { "ET" , 231, "014", URGN_TERRITORY, "002" },
@@ -195,7 +192,6 @@ static KnownRegion knownRegions[] = {
     { "HR" , 191, "039", URGN_TERRITORY, "150" },
     { "HT" , 332, "029", URGN_TERRITORY, "019" },
     { "HU" , 348, "151", URGN_TERRITORY, "150" },
-    { "IC" ,  -1, "015", URGN_TERRITORY, "002" },
     { "ID" , 360, "035", URGN_TERRITORY, "142" },
     { "IE" , 372, "154", URGN_TERRITORY, "150" },
     { "IL" , 376, "145", URGN_TERRITORY, "142" },
diff --git a/icu/icu4c/source/test/intltest/regiontst.cpp b/icu/icu4c/source/test/intltest/regiontst.cpp
index d8a6723bfe353e71d5a82102a09d60c640b14071..bbee1863639157a6a8b7f3f722a882754e0d7c14 100644
--- a/icu/icu4c/source/test/intltest/regiontst.cpp
+++ b/icu/icu4c/source/test/intltest/regiontst.cpp
@@ -122,17 +122,14 @@ static KnownRegion knownRegions[] = {
     { "CZ" , 203, "151", URGN_TERRITORY, "150" },
     { "DD" , 276, "155", URGN_TERRITORY, "150" },
     { "DE" , 276, "155", URGN_TERRITORY, "150" },
-    { "DG" , -1 , "QO" , URGN_TERRITORY, "009" },
     { "DJ" , 262, "014", URGN_TERRITORY, "002" },
     { "DK" , 208, "154", URGN_TERRITORY, "150" },
     { "DM" , 212, "029", URGN_TERRITORY, "019" },
     { "DO" , 214, "029", URGN_TERRITORY, "019" },
     { "DZ" ,  12, "015", URGN_TERRITORY, "002" },
-    { "EA" ,  -1, "015", URGN_TERRITORY, "002" },
     { "EC" , 218, "005", URGN_TERRITORY, "019" },
     { "EE" , 233, "154", URGN_TERRITORY, "150" },
     { "EG" , 818, "015", URGN_TERRITORY, "002" },
-    { "EH" , 732, "015", URGN_TERRITORY, "002" },
     { "ER" , 232, "014", URGN_TERRITORY, "002" },
     { "ES" , 724, "039", URGN_TERRITORY, "150" },
     { "ET" , 231, "014", URGN_TERRITORY, "002" },
@@ -169,7 +166,6 @@ static KnownRegion knownRegions[] = {
     { "HR" , 191, "039", URGN_TERRITORY, "150" },
     { "HT" , 332, "029", URGN_TERRITORY, "019" },
     { "HU" , 348, "151", URGN_TERRITORY, "150" },
-    { "IC" ,  -1, "015", URGN_TERRITORY, "002" },
     { "ID" , 360, "035", URGN_TERRITORY, "142" },
     { "IE" , 372, "154", URGN_TERRITORY, "150" },
     { "IL" , 376, "145", URGN_TERRITORY, "142" },
